src/main/java/com/exasol/exasoltestsetup/standalone/ConnectionDetailProvider.java

```diff
@@ -0,0 +1,42 @@
+package com.exasol.exasoltestsetup.standalone;
+
+/**
+ * This interface provides connection details for running Exasol database.
+ */
+interface ConnectionDetailProvider {
+
+    /**
+     * Get the IP address of the management node.
+     * 
+     * @return ip address
+     */
+    public String getManagementNodeIp();
```

Comment:
Why not using [INetAddress](https://docs.oracle.com/javase/7/docs/api/java/net/InetAddress.html) instead of String.

Reply:
Or https://docs.oracle.com/javase/7/docs/api/java/net/InetSocketAddress.html (IP + ssh Port)

Reply:
I also want to support host names.
I renamed the method to make that clear.